hooks/useFeeds.ts

```
@@ -0,0 +1,61 @@
+import delay from "@/lib/delay";
+import { Collection, Token } from "@/types/token";
+import { useCallback, useEffect, useState } from "react";
+
+const fetchTokens = async (collectionAddresses: Collection[]) => {
+  while (1) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)